apps/SKonsole/Commands/PromptChatCommand.cs

```diff
@@ -63,32 +64,57 @@ private static async Task RunPromptChat(CancellationToken token, ILogger logger)
 
     private static async Task RunChat(IKernel kernel, ILogger? logger, ISKFunction chatFunction)
     {
+        AnsiConsole.MarkupLine("[grey]Press Enter twice to send a message.[/]");
+        AnsiConsole.MarkupLine("[grey]Enter 'exit' to exit.[/]");
         var contextVariables = new ContextVariables();
 
-        var history = "";
+        var history = string.Empty;
         contextVariables.Set("history", history);
 
         var botMessage = await kernel.RunAsync(contextVariables, chatFunction);
         var userMessage = string.Empty;
 
-        var log = logger ?? kernel.LoggerFactory.CreateLogger<PromptChatCommand>();
+        void HorizontalRule(string title, string style = "white bold")
+        {
+            AnsiConsole.WriteLine();
+            AnsiConsole.Write(new Rule($"[{style}]{title}[/]").RuleStyle("grey").LeftJustified());
+            AnsiConsole.WriteLine();
+        }
 
         while (userMessage != "exit")
         {
-            var botMessageFormatted = "\nAI: " + botMessage.ToString() + "\n";
-            log.LogInformation("{botMessage}", botMessageFormatted);
-            log.LogInformation(">>>");
+            HorizontalRule("AI", "green bold");
+            AnsiConsole.Foreground = ConsoleColor.Green;
+            AnsiConsole.WriteLine(botMessage.ToString());
+            AnsiConsole.ResetColors();
 
+            HorizontalRule("User");
             userMessage = ReadMultiLineInput();
+
             if (userMessage == "exit")
             {
                 break;
             }
 
-            history += $"{botMessageFormatted}Human: {userMessage}\nAI:";
+            history += $"AI: {botMessage}\nHuman: {userMessage} \n";
             contextVariables.Set("history", history);
 
-            botMessage = await kernel.RunAsync(contextVariables, chatFunction);
+            botMessage = await AnsiConsole.Progress()
+                .AutoClear(true)
+                .Columns(new ProgressColumn[]
+                {
+                    new TaskDescriptionColumn(),
+                    new SpinnerColumn(),
+                })
+                .StartAsync(async ctx =>
```

Comment:
Love this. Should refactor this for commands to share eventually as appropriate. For now, I'm copying :)